src/Pages/OpenSoon/OpenSoon.js

```diff
@@ -1,8 +1,57 @@
 import React, { Component } from "react";
+import OpenSoonClass from "./OpenSoonClass";
+import FirstModal from "../OpenSoon/OpenSoonClass";
+import "./OpenSoon.scss";
+
+class OpenSoon extends Component {
+  constructor() {
+    super();
+    this.state = {
+      classList: {},
+    };
+  }
+
+  componentDidMount = () => {
+    fetch("/data/ClassListData.json")
+      .then((res) => res.json())
+      .then((res) => this.setState({ classList: res }));
+  };
 
-export class OpenSoon extends Component {
   render() {
-    return <div></div>;
+    const { classList } = this.state;
+    const { sortClass, sortPrice } = this;
+    // console.log("첫번째 콘솔", this.state.classList);
+    return (
+      <>
+        <div className="banner">
+          <div className="boxContainer">
+            <div className="firstBox">클래스를 응원하면 더 빨리 제작돼요</div>
+
+            <div className="secondBox">응원한 클래스</div>
+            <div className="thirdBox">내 응원권</div>
+          </div>
+        </div>
+        <filter sortClass={sortClass} sortPrice={sortPrice} />
+        <div className="openSoonMain">
+          <div className="list">
+            {classList.pending_classes &&
+              classList.pending_classes.map((e) => {
```

Comment:
인자의 값을 e보다는 하나의 요소를 의미하는 직관적인 이름을 작성해주시는게 좋습니다. 
```suggestion
              classList.pending_classes.map((class) => {
```

Reply:
네 수정하였습니다!!!